pkgs/development/compilers/rust/default.nix

```diff
@@ -53,12 +53,16 @@ in
       bootstrapRustPackages = self.buildRustPackages.overrideScope' (_: _:
         lib.optionalAttrs (stdenv.buildPlatform == stdenv.hostPlatform)
           (selectRustPackage buildPackages).packages.prebuilt);
-      bootRustPlatform = makeRustPlatform bootstrapRustPackages;
+      bootRustPlatform = makeRustPlatform {
+        inherit (bootstrapRustPackages) cargo rustc;
+      };
     in {
       # Packages suitable for build-time, e.g. `build.rs`-type stuff.
       buildRustPackages = (selectRustPackage buildPackages).packages.stable;
       # Analogous to stdenv
-      rustPlatform = makeRustPlatform self.buildRustPackages;
+      rustPlatform = makeRustPlatform {
+        inherit (self.buildRustPackages) cargo rustc;
```

Comment:
should we include cargo-auditable here? are there any cross-compilation implications?

same question for the one below

Reply:
Ah, maybe.  I'll look into it.